packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,144 @@
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-select slot="end">
+                <fast-option value="1">Option 1</fast-option>
+                <fast-option value="2">Option 2</fast-option>
+                <fast-option value="3">Option 3</fast-option>
+            </fast-select>
+                Accordion one content
+            <div slot="heading">Accordion one</div>
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-number-field placeholder="number" slot="end"></fast-number-field>
+                Accordion two content
+            <div slot="heading">Accordion two</div>
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-select slot="end">
+                <fast-option value="1">Option 1</fast-option>
+                <fast-option value="2">Option 2</fast-option>
+                <fast-option value="3">Option 3</fast-option>
+            </fast-select>
+            Accordion three content<div slot="heading">Accordion three</div>
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-number-field placeholder="number" slot="end"></fast-number-field>
+                Accordion four content
+            <div slot="heading">Accordion four</div>
+    </fast-accordion-item>
+</fast-accordion>
+```
+
+### API
+
+#### FAST Accordion
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_ Unchanged
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+#### FAST Accordion Item
+
+[FAST accordion-item API](https://github.com/microsoft/fast/tree/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion-item)
+
+-   _Component Name:_ `nimble-accordion-item`
+-   _Properties/Attributes:_
+    -   `appearance` - `block`, `outline`, `ghost`
+    -   `error-visible` - boolean
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_
+    -   `appearance` - `block`, `outline`, `ghost`
+    -   `error-visible` - boolean
+-   _Slots:_ Unchanged
+
+The Figma design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` buttons. The nimble-button pattern with these styles seems to fit the styling of the accordion, so these styles can be used. The attributes of `block`, `outline`, and `ghost` will most likely be implemented with the name `appearance`. A few styling elements like the background color change on click that is used in the button will have to be removed.
```

Comment:
Something to keep in mind for this implementation direction. If you end up in a position where you include the shared button styles and then have to override a significant number of them that would be a big drawback for maintainability. In order to understand the styles you now only have to look at the button styles but then keep in your head how they need to be overridden.

What I want to avoid is if you open devtools and inspect the styles and see tons of constantly overridden styles then it is probably not a good fit for the styles to be shared.

An extreme (somewhat misleading) example of styles being overridden excessively:
![image](https://github.com/ni/nimble/assets/1588923/ffd8653c-e8c6-4826-b532-0ac706c46281)

It's not that overriding styles is bad. Overriding styles is expected for things like state changes from the default, i.e. override the border on hover. So I'm not discouraging you from finding good patterns to share styles, it's been very valuable for buttons, etc, just making sure you are aware of what a concern may be in finding the right styles to share.

Reply:
I talked with Mert about this and took another look at the Figma spec, and we decided to not use the button pattern styles. This should solve this issue of overriding too many styles, I'll update the spec to reflect this decision.